build.gradle

```diff
@@ -320,6 +320,8 @@ project(':iceberg-flink') {
     compile project(':iceberg-parquet')
     compile project(':iceberg-hive-metastore')
 
+    // for dropwizard histogram metrics implementation
+    compileOnly "org.apache.flink:flink-metrics-dropwizard"
```

Comment:
I was wondering if this should be `compile` dep. flink-metrics-dropwizard is not part of flink-dist. User would need to bring it in. I stayed with `compileOnly` as the same argument applied to other `flink-table-*` deps
```
    compileOnly "org.apache.flink:flink-table-api-java-bridge_2.12"
    compileOnly "org.apache.flink:flink-table-planner-blink_2.12"
    compileOnly "org.apache.flink:flink-table-planner_2.12"
```

Reply:
I downloaded the  [flink-1.13.2](https://flink.apache.org/downloads.html#apache-flink-1132) from the apache flink official website.  And checked the `dropwizard` related class: 

```bash
➜  flink-1.13.2 grep -R DropwizardMeterWrapper . 
Binary file ./plugins/metrics-graphite/flink-metrics-graphite-1.13.2.jar matches
```

Looks like the jar was included in the `metrics-graphite` plugins,  which means we could not load the classes from flink dist binary by default, unless we enable the metrics-graphite plugin.  The means we couldn't run the iceberg flink sink successfully unless we enable the metrics-graphite plugin,  that's quite unfriendly for the quickstart. 

If we really need the dropwizad jar , then I will suggest to include into the iceberg-flink-runtime.jar, so that we could get rid of the above issue.

Reply:
included the dropwizard jar in the iceberg-flink-runtime.jar. also removed the unnecessary exclude for flink in the iceberg-flink-runtime in build.gradle, as all flink deps are marked as `compileOnly`. Verified that we don't need the exclude with a locally built iceberg-flink-runtime.jar